controllers/administration.js

```diff
@@ -1701,8 +1706,14 @@ const renderClassEdit = (req, res, next) => {
 								currentClass.keepYear = true;
 							}
 						}
+
+						if (currentClass.year) {
+							isUpgradable = (lastDefinedSchoolYearId !== (currentClass.year || {}))
```

Comment:
I think a check like this should also be done on the main classes-overview page. There you can still click upgrade on classes in the last year.